js/controller.js

```diff
@@ -1313,6 +1319,15 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
       this.trySetAnamorphicFixState(true);
       // In case ad was skipped or errored while stalled
       this.setBufferingState(false);
+
```

Comment:
@KseniiaSokolovaEpam can we remove extra line here?

Reply:
\+